go/Dockerfile

```diff
@@ -10,8 +10,8 @@ RUN go build -o quickstart
 FROM gcr.io/distroless/base-debian10
 #FROM gcr.io/distroless/base-debian10:debug
 COPY --from=build /opt/src/go/quickstart /
-COPY static/ /static
-COPY html/ /html
+# COPY static/ /static
+# COPY html/ /html
```

Comment:
delete? looks like there is some commented code in `server.go` that needs to be deleted too

Reply:
done